From 336773f36d0642c58724af0fed736ce9f454c564 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 9 Jan 2018 16:07:11 +0800
Subject: [PATCH] ARM: dts: rockchip: rk3036: Add vpu qos node

The vpu qos registers need to save and restore when reset.

Change-Id: I649cf4a360842ad1abb06c35a6fd8d3868fbf706
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm/boot/dts/rk3036.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/rk3036.dtsi b/arch/arm/boot/dts/rk3036.dtsi
index 6dc6696c5641..44354bf5e477 100644
--- a/arch/arm/boot/dts/rk3036.dtsi
+++ b/arch/arm/boot/dts/rk3036.dtsi
@@ -346,6 +346,11 @@
 		status = "disabled";
 	};
 
+	qos_vpu: qos@1012e000 {
+		compatible = "syscon";
+		reg = <0x0 0x1012e000 0x0 0x20>;
+	};
+
 	gic: interrupt-controller@10139000 {
 		compatible = "arm,gic-400";
 		interrupt-controller;
@@ -505,6 +510,7 @@
 				clocks = <&cru ACLK_VCODEC>,
 					 <&cru HCLK_VCODEC>,
 					 <&cru ACLK_HEVC>;
+				pm_qos = <&qos_vpu>;
 			};
 
 		};
-- 
2.35.3

